geany.appdata.xml.in

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
Why is this one file in the entire source not GPL?   I suspect LICENSE needs updating.